models/tokens/ethereum/tokens_ethereum_transfers_base.sql

```diff
@@ -0,0 +1,22 @@
+{{ config(
+        schema = 'tokens_ethereum',
+        alias = 'transfers_base',
+        partition_by = ['token_standard', 'block_date'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+        unique_key = ['block_number', 'tx_index', 'evt_index', 'trace_address'],
+        )
+}}
+
+-- TODO: use variable for native_contract_address
+{{transfers_base(
+    blockchain='ethereum',
+    traces = source('ethereum','traces'),
+    transactions = source('ethereum','transactions'),
+    erc20_transfers = source('erc20_ethereum','evt_Transfer'),
+    native_contract_address = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee',
+    wrapped_token_deposit = source('zeroex_ethereum', 'weth9_evt_deposit'),
```

Comment:
also — weth isn't connected to zeroex. it'd be better to resubmit weth contract to Dune and change the source.

Reply:
fun fact: The first WETH contract was actually first developed and deployed by some people from 0x labs.
Agree though it should be renamed in the future, but it has built up a lot of downstream dependencies so name changes are not trivial.